corehq/apps/export/models/new.py

```diff
@@ -476,6 +503,7 @@ def __insert_system_properties(cls, table, properties, top=True):
 
         for static_column in properties:
             index, existing_column = table.get_column(static_column.item.path, static_column.item.transform)
+            (existing_column or static_column).extra_data(**data)
```

Comment:
don't love this, but it's the only way to get more data into these columns like domain and repeat